airbyte-integrations/base-python/Dockerfile

```
@@ -1,22 +1,15 @@
 FROM python:3.7-slim
 COPY --from=airbyte/integration-base:dev /airbyte /airbyte
 
-WORKDIR /airbyte
-ENV VIRTUAL_ENV=/airbyte/env
-RUN python -m venv $VIRTUAL_ENV
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-
-WORKDIR /airbyte/airbyte_protocol
-COPY airbyte_protocol .
+WORKDIR /airbyte/base_python_code
```

```suggestion
WORKDIR /airbyte/base-python
```

Can we make this match the source directory name?

`_code` doesn't seem like a useful distinction when we're copying the code over in the next step.

---

Actually it is dangerous to do that, it hit me. It was trying to load the module from the local path instead of the installed version.